src/Compilers/CSharp/Portable/CSharpCodeAnalysis.csproj

```diff
@@ -68,6 +68,7 @@
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.CompilerServer" />
     <InternalsVisibleTo Include="VBCSCompiler" />
     <InternalsVisibleTo Include="VBCSCompilerPortable" />
+    <InternalsVisibleToTest Include="Roslyn.Compilers.CompilerServer.UnitTests" />
```

Comment:
@dotnet/roslyn-compiler Do we need any special approval for IVT for test projects?